src/main/java/com/minecolonies/blockout/views/ScrollingView.java

```diff
@@ -15,7 +15,7 @@
     protected ScrollingContainer container;
     protected Scrollbar          scrollbar;
     //  Params
-    private int scrollbarWidth = DEFAULT_SCROLLBAR_WIDTH;
+    private final int scrollbarWidth = DEFAULT_SCROLLBAR_WIDTH;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1170)